enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -109,6 +110,33 @@ def generate_payload(df, logbook_header_id):
         for date in skipped_dates:
             print(date)
 
+    get_logBook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
+
+    emptyCounter = 0
+    dateList = {}
+    date = 0
+    for days in json.loads(get_logBook_response.text)["data"]:
+        if days["id"] == "00000000-0000-0000-0000-000000000000":
+            dateList[date] = days["date"]
+            emptyCounter = emptyCounter + 1
+        date = date + 1
+
+    print(dateList)
+    if emptyCounter != 0:
```

Comment:
You can do this instead, to check whether empty_dates are empty or not
```suggestion
    if empty_dates:
```